internal/core/thirdparty/knowhere/CMakeLists.txt

```diff
@@ -12,8 +12,8 @@
 #-------------------------------------------------------------------------------
 
 # Update KNOWHERE_VERSION for the first occurrence
-set( KNOWHERE_VERSION 20a6289 )
-set( GIT_REPOSITORY  "https://github.com/zilliztech/knowhere.git")
+set( KNOWHERE_VERSION template_data )
+set( GIT_REPOSITORY  "https://github.com/cqy123456/zilliztech-knowhere.git")
```

Comment:
Once the pull request from knowhere is merged, please ensure that the GIT_REPOSITORY is redirected to the right repository